### PR TITLE
Touch up a few files

### PR DESCRIPTION
**Pull request by penev92**
Just a small bunch of simple code touch-ups.

### CODE REVIEW

**Comment by Phrohdoh** [comment]
Why are you removing `ProximityTrigger.Id`?

File: OpenRA.Game/Traits/World/ActorMap.cs [hunk 1]
```diff
@@ -63,13 +61,13 @@ public CellTrigger(int id, CPos[] footprint, Action<Actor> onActorEntered, Actio
...
 
-			public void Tick(ActorMap am)
+			public void Tick(ActorMap actorMap)
 			{
 				if (!Dirty)
 					return;
 
-				var oldActors = currentActors;
-				currentActors = Footprint.SelectMany(c => am.GetUnitsAt(c)).ToList();
+				var oldActors = currentActors.ToList();
```
Comment (pchote) [inline OpenRA.Game/Traits/World/ActorMap.cs]: This forces a(n unnecessary) copy of the list.  More than just a simple touchup!

File: OpenRA.Game/Traits/World/ActorMap.cs [hunk 2]
```diff
@@ -129,7 +123,7 @@ public void Tick(ActorMap am)
 				if (!Dirty)
 					return;
 
-				var oldActors = currentActors;
+				var oldActors = currentActors.ToList();
```
Comment (pchote) [inline OpenRA.Game/Traits/World/ActorMap.cs]: This forces a(n unnecessary) copy of the list. More than just a simple touchup!

**Comment by penev92** [comment]
Reverted the `.ToList()`s.

**Comment by Mailaender** [comment]
:+1:

**Comment by penev92** [comment]
@Phrohdoh Because it's unused. Maybe a leftover from something old?

File: OpenRA.Game/Traits/World/ActorMap.cs [hunk 3]
```diff
@@ -88,24 +86,20 @@ public void Tick(ActorMap am)
 
 		class ProximityTrigger : IDisposable
 		{
-			public readonly int Id;
-			public WPos Position { get; private set; }
-			public WDist Range { get; private set; }
```
Comment (pchote) [inline OpenRA.Game/Traits/World/ActorMap.cs]: This is useful info that makes sense to be public, even if nothing is using it right now.

Comment (Phrohdoh) [inline OpenRA.Game/Traits/World/ActorMap.cs]: You and @Mailaender have told me many times not to make something public unless it is being used by something.

Comment (pchote) [inline OpenRA.Game/Traits/World/ActorMap.cs]: That's a valid point I guess.

**Comment by pchote** [comment]
Looks like a reasonable set of code cleanups :+1:

**Pull request merged**